lib/styles/settings/_buttons.scss

```diff
@@ -16,6 +17,402 @@ $button-ripple-alpha: 0.16;
+			'color': $color-default-text-inverted,
+			'icon': $color-default-icon-inverted,
+			'background': $color-info-background-strong,
+			'background-hovered': $color-info-background-strong-hovered,
+			'background-focused': $color-info-background-strong-focused,
+			'background-disabled': $color-info-background-strong-disabled,
+			'ripple': $color-default-ripple-inverted,
+			'disabled': (
+				'color': $color-default-text-inverted,
+				'icon': $color-default-icon-inverted,
+				'background': $color-info-background-strong-disabled,
+			),
+		),
+		'outlined': (
+			'color': $color-info-text,
+			'icon': $color-info-icon,
+			'border': $color-info-border,
+			'background': $color-info-background-ghost,
+			'background-hovered': $color-info-background-ghost-hovered,
+			'background-focused': $color-info-background-ghost-focused,
+			'background-disabled': $color-info-background-ghost-disabled,
+			'ripple': $color-info-ripple,
+			'disabled': (
+				'color': $color-info-text,
+				'border': $color-info-border-disabled,
+				'icon': $color-info-icon-disabled,
+				'background': '',
+			),
+		),
+		'text': (
+			'color': $color-info-text,
+			'icon': $color-info-icon,
+			'background': $color-info-background-ghost,
+			'background-hovered': $color-info-background-ghost-hovered,
+			'background-focused': $color-info-background-ghost-focused,
+			'background-disabled': $color-info-background-ghost-disabled,
+			'disabled': (
+				'color': $color-info-text,
+				'icon': $color-info-icon-disabled,
+				'ripple': $color-info-ripple,
+			),
+		),
+	),
+);
+
+$button-colors: (
+	'primary': (
+		'filled': (
+			'color': $color-default-text-inverted,
+			'color-hovered': $color-primary-text-hovered,
+			'color-focused': $color-primary-text-focused,
+			'icon': $color-default-icon-inverted,
+			'background': $color-primary-background-strong,
+			'background-hovered': $color-primary-background-strong-hovered,
+			'background-focused': $color-primary-background-strong-focused,
+			'background-disabled': $color-primary-background-strong-disabled,
+			'ripple': $color-default-ripple-inverted,
+			'disabled': (
+				'color': $color-default-text-inverted,
+				'icon': $color-default-icon-inverted,
+				'background': $color-primary-background-strong-disabled,
+			),
+		),
+		'outlined': (
+			'color': $color-primary-text,
+			'color-hovered': $color-primary-text-hovered,
+			'color-focused': $color-primary-text-focused,
+			'icon': $color-primary-icon,
+			'border': $color-primary-border,
+			'background': $color-primary-background-ghost,
+			'background-hovered': $color-primary-background-ghost-hovered,
+			'background-focused': $color-primary-background-ghost-focused,
+			'background-disabled': $color-primary-background-ghost-disabled,
+			'ripple': $color-primary-ripple,
+			'disabled': (
+				'color': $color-primary-text-disabled,
+				'border': $color-primary-border-disabled,
+				'icon': $color-primary-icon-disabled,
+				'background': 'transparent',
```

Comment:
why do we sometimes have `''` and sometimes `'transparent'` as a `background-color`?

Reply:
I replaced all `transparent` to correct token f.e.: `$color-primary-background-ghost-disabled'`